templates/snap-details.html

```diff
@@ -28,7 +28,7 @@ <h1>{{ snap_title }}</h1>
 
             <p class="p-media-object__content--large">
               {% if is_linux %}
-                <a class="p-button--positive" href="snap://{{ package_name }}">Install</a>
+                <a class="p-button--positive" href="snap://{{ package_name }}" onclick="ga('gtm1.send', {hitType: 'event', eventCategory: 'Snap details', eventAction: 'Clicked Install button', eventLabel: 'Install {{ package_name }}'})">Install</a>
```

Comment:
When we have more places where we would send custom events it would be good to abstract it somehow to avoid inline click events, but it's fine for now.